core/templates/pages/community-dashboard-page/services/translate-text.service.spec.ts

```diff
@@ -0,0 +1,130 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Tests for translate-text service.
+ */
+
+// TODO(#7222): Remove the following block of unnnecessary imports once
+// UserService.ts is upgraded to Angular 8.
+import { UpgradedServices } from 'services/UpgradedServices';
+// ^^^ This block is to be removed.
+
+require(
+  'pages/community-dashboard-page/services/translate-text.service.ts');
+
+describe('TranslateTextService', function() {
+  let TranslateTextService;
+  let $httpBackend;
+
+  beforeEach(angular.mock.module('oppia'));
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    var ugs = new UpgradedServices();
```

Comment:
No upgraded services please... You can use provide instead. Adding @srijanreddy98 for advice.

Reply:
@nithusha21 Hmm. I don't think the UpgradedServices is necessary here actually. Removed.